docs/post-call-survey.md

```diff
@@ -2,6 +2,41 @@
 
 You can configure post-call surveys to collect valuable feedback like quality of services or net promoter score after a call ends.
 
+## Create a 1-question poll
```

Comment:
This file will be removed from this PR right?

Reply:
Yes